.ddev/.homeadditions/appserver_build.sh

```diff
@@ -1,36 +1,36 @@
 #!/usr/bin/env bash
 
 # Variables to indicate key settings files or directories for Drupal.
-DRUPAL_ROOT=/app/web
+DRUPAL_ROOT=/var/www/html/web
 DRUPAL_SETTINGS_FILE=$DRUPAL_ROOT/sites/default/settings.php
 DRUPAL_SERVICES_FILE=$DRUPAL_ROOT/sites/default/services.yml
 DRUPAL_CUSTOM_CODE=$DRUPAL_ROOT/modules/custom
 DRUPAL_CUSTOM_THEME=$DRUPAL_ROOT/themes/custom/nicsdru_nidirect_theme
```

Comment:
I had removed these unused variables. (DRUPAL_SETTINGS_FILE onward)

Reply:
Good spot. Had this open in another IDE which doesn't highlight unused vars :)